src/components/home/Home.jsx

```diff
@@ -42,35 +50,37 @@ export default function Home() {
 
   return (
     <>
+      <div className='homeContainer'>
+        <h1>TropiCafe</h1>
+        <button className='addShopButton' onClick={(event) => { toggleAddShopModal(event, true); }}>Add Shop</button>
 
-      <button onClick={(event) => { toggleAddShopModal(event, true); }}>Add Shop</button>
-
-      <div className='popularShopsContainer'>
-        {seeMorePopular ? <>
-          <h2>Popular Coffee Shops</h2>
-          <ShopList ShopList={allPopularShops} />
-          <button onClick={(event) => { toggleMorePopular(event, false); }}>See Less</button>
-        </> : <>
-          <h2>Popular Coffee Shops</h2>
-          <ShopList ShopList={popularShops} />
-          <button onClick={(event) => { toggleMorePopular(event, true); }}>See More Popular Shops</button>
-        </>}
-      </div>
+        <div className='popularShopsContainer'>
+          {seeMorePopular ? <>
+            <h2 className='popular-title'>Popular Coffee Shops</h2>
+            <ShopList ShopList={allPopularShops} />
+            <button onClick={(event) => { toggleMorePopular(event, false); }}>See Less</button>
```

Comment:
You can use `setSeeMoreRecent` here. The `button` element doesn't have any default action.

```
onClick={() => setSeeMoreRecent(false)}
```